### PR TITLE
Pr/consoleobserver

### PR DESCRIPTION
**Pull request by kbilsted**

### CODE REVIEW

File: src/Okanshi/Observer.fs [hunk 1]
```diff
@@ -49,3 +49,30 @@ type MemoryMetricObserver(poller : IMetricPoller, numberOfSamplesToStore) as sel
     
     interface IProcessingMetricObserver with
         member self.GetObservations() = self.GetObservations()
+
+/// Observer that prints to the Console. An easy way to see that Okanshi is working
+/// The serializer function could be e.g. a standard serializer such as newtonsoft Json: 
+/// <code>observer = new ConsoleObserver(poller, x => JsonConvert.SerializeObject(x, Formatting.Indented))</code>
```
Comment (kichristensen) [inline src/Okanshi/Observer.fs]: I think it would make better sense to describe using console write line instead

File: src/Okanshi/Observer.fs [hunk 2]
```diff
@@ -49,3 +49,30 @@ type MemoryMetricObserver(poller : IMetricPoller, numberOfSamplesToStore) as sel
...
+type ConsoleObserver(poller : IMetricPoller, serializer : Func<Object, string>) as self = 
+    
+    do
+        poller.RegisterObserver(new Func<Metric seq, Task>(self.Update))
+   
+    new (serializer : Func<Object, string>) =
```
Comment (kichristensen) [inline src/Okanshi/Observer.fs]: The other observer doesn't use default registry by default, I think the behaviour should be the same

File: src/Okanshi/Observer.fs (hunk 1 above)
Comment (kbilsted) [reply src/Okanshi/Observer.fs]: ah you want the tutorial updated

File: src/Okanshi/Observer.fs (hunk 2 above)
Comment (kichristensen) [reply src/Okanshi/Observer.fs]: We have talked a lot the problems with `OkanshiMonitor`, I don't think we should make the same mistake here

File: src/Okanshi/Observer.fs (hunk 1 above)
Comment (kichristensen) [reply src/Okanshi/Observer.fs]: Not really. Why do this comment, which is part of the API documentation, reference JsonConvert? I would expect something else based on the name of the observer.

File: src/Okanshi/Observer.fs (hunk 2 above)
Comment (kbilsted) [reply src/Okanshi/Observer.fs]: I'll remove the ctor

**Pull request merged**